doc/release-notes/rl-2511.section.md

```diff
@@ -139,6 +139,8 @@
 
 - `services.forgejo.dump.age` now defaults to `4w`, which deletes dumps older than 4 weeks. This new behaviour could result in older backups being deleted.
 
+- `services.journald.gateway.user` and `services.journald.gateway.system` now defaults to `false`. This new behaviour matches the default behaviour of the [`systemd-journal-gatewayd`](https://www.freedesktop.org/software/systemd/man/latest/systemd-journal-gatewayd.service.html) service itself.
+
```

Comment:
This needs to be in the NixOS manual in `nixos/doc/manual/release-notes/rl-2511.section.md`, not the Nixpkgs manual in `doc/release-notes/rl-2511.section.md`

Reply:
Thanks, I moved it.